src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 20;
+    private static final long VALID_ID = 123883022L;
+    private static final String VALID_LOGIN = "UserName";
+    private static final String VALID_PASSWORD = "BestPasswordEver123";
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user = new User();
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(VALID_AGE);
+        user.setId(VALID_ID);
+        user.setLogin(VALID_LOGIN);
+        user.setPassword(VALID_PASSWORD);
+    }
+
+    @Test
+    void register_lowerThanExpectedAge_NotOk() {
+        int age = 17;
+        user.setAge(age);
+        assertTrue(user.getAge() < 18);
+    }
```

Comment:
The test `register_lowerThanExpectedAge_NotOk` only checks the age condition without calling the `register` method. To properly test this case, you should invoke `registrationService.register(user)` and assert that a `RegistrationException` is thrown.